### PR TITLE
CRT-496 Update Dependencies

### PR DESCRIPTION
**Pull request by akaragachev**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
        },
     "require": {
         "php": "^8.0",
-        "lanfix/goaop-framework": "^4.0",
+        "lanfix/goaop-framework": "4.0.4",
```
Comment (Wernke96) [inline composer.json]: Don't we want to use our package here?

**Review by Wernke96 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (akaragachev) [reply composer.json]: I'm going to re-route it in core using vcs repositories:
image

Comment (Wernke96) [reply composer.json]: I think we should reroute the goap-framework here. Since this is where the package comes in.

File: composer.json [hunk 2]
```diff
@@ -14,9 +14,9 @@
        },
     "require": {
         "php": "^8.0",
-        "lanfix/goaop-framework": "^4.0",
+        "sourcetoad/goaop-framework": "4.0.4",
```
Comment (Wernke96) [inline composer.json]: if you try to composer install this it doesn't work

**Review by Wernke96 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (frankspress) [reply composer.json]: Also please run composer install after upping any dependency so that the composer.lock is updated and added to the commit.

Comment (Wernke96) [reply composer.json]: I got this to work, and here is my composer.json file.
Screenshot 2024-06-06 at 12 59 14 PM

Comment (akaragachev) [reply composer.json]: Updated

**Review by Wernke96 — APPROVED** [review]

**Review by frankspress — APPROVED** [review]

**Pull request merged**